Makefile

```
@@ -1,6 +1,17 @@
+DIR := ${CURDIR}
+
 install-dependencies:
 	sudo dnf -y install python3-flask npm
 	npm install
 
 run:
-	FLASK_ENV=development python3 main.py
+	FLASK_ENV=development FLASK_APP=packit_dashboard.app flask run
+
+run-docker:
+	docker run -p 443:8443 -v $(DIR)/secrets:/secrets -i usercont/packit-dashboard:stg
```

Can you define the image name as a variable at the top of the `Makefile`:
```
IMAGE ?= usercont/packit-dashboard:stg
```

and then

```suggestion
	docker run -p 443:8443 -v $(DIR)/secrets:/secrets -i $(IMAGE)
```

---

fixed